devnet/docker/icon-bsc/scripts/config/arctic.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
```

Comment:
renamed the arctic.sh to icon-ice-arctic-config.sh
did not use the name lisbon because it is the name of ICON testnet but the config holds parameters for ICON mainnet as well.